components/vc-notification/Notification.jsx

```diff
@@ -67,7 +67,7 @@ const Notification = {
     remove(key) {
       this.setState(previousState => {
         return {
-          notices: previousState.notices.filter(notice => notice.key !== key),
+          notices: previousState.notices.filter(notice => notice.key !== key && notice.updateKey !== key),
```

Comment:
You are right!
But maybe we should change 
`const close = createChainedFunction(remove.bind(this, key), onClose);`
to
`const close = createChainedFunction(remove.bind(this, notice.key), onClose);`
is more better solution.